deepchecks/utils/abstracts/feature_drift.py

```diff
@@ -0,0 +1,185 @@
+                max_num_categories_for_drift=self.max_num_categories_for_drift,
+                min_category_size_ratio=self.min_category_size_ratio,
+                max_num_categories_for_display=self.max_num_categories_for_display,
+                show_categories_by=self.show_categories_by,
+                numerical_drift_method=self.numerical_drift_method,
+                categorical_drift_method=self.categorical_drift_method,
+                ignore_na=self.ignore_na,
+                min_samples=self.min_samples,
+                with_display=with_display,
+                dataset_names=(test_dataframe_name, train_dataframe_name)
+            )
+
+            if value == 'not_enough_samples':
+                not_enough_samples.append(column_name)
+                value = None
+            else:
+                plots[column_name] = display
+
+            results[column_name] = {
+                'Drift score': value,
+                'Method': method,
+                'Importance': (
+                    feature_importance[column_name]
+                    if feature_importance is not None
+                    else None
+                )
+            }
+
+        if len(not_enough_samples) == len(results.keys()):
+            raise NotEnoughSamplesError(
+                f'Not enough samples to calculate drift score. Minimum {self.min_samples} samples required. '
+                'Note that for numerical columns, None values do not count as samples.'
+                'Use the \'min_samples\' parameter to change this requirement.'
+            )
+
+        if not with_display:
+            return results, []
+
+        if self.sort_feature_by == 'feature importance' and features_order is not None:
+            sorted_by = self.sort_feature_by
+            features_order = [feat for feat in features_order if feat in results]
+            columns_order = features_order[:self.n_top_columns]
+        elif self.sort_feature_by == 'drift + importance' and features_order is not None:
+            sorted_by = 'the sum of the drift score and the feature importance'
+            feature_columns = [feat for feat in features_order if feat in results]
+            key = lambda col: (results[col]['Drift score'] or 0) + results[col]['Importance']
+            columns_order = sorted(feature_columns, key=key, reverse=True)[:self.n_top_columns]
+        else:
+            sorted_by = 'drift score'
+            key = lambda col: results[col]['Drift score'] or 0
+            columns_order = sorted(results.keys(), key=key, reverse=True)[:self.n_top_columns]
+
+        headnote = [
+            textwrap.dedent(
+                f"""
+                <span>
+                The Drift score is a measure for the difference between two distributions, in this check - the test
+                and train distributions.<br> The check shows the drift score and distributions for the features, sorted
```

Comment:
replace features with a argument received externally stating wether it is feature or property - 
same as data_table_name argument in validate_length_and_calculate_column_types

Reply:
fixed